### ISSUE TITLE
question: 该项目会考虑增加更多的语言支持吗？例如阿拉伯语

### ISSUE THREAD

**Issue by JunwuHuang** [apitable/apitable#1669]
增加更多的语言支持有利于发展潜在客户，同时也能让有兴趣的开发者参与共建。

**Comment by mr-kelly** [comment]
Yes, APITable supports internationalization (i18n). However, Arabic is an RTL (Right-to-Left) language, which presents a significant technical challenge for our frontend UI layout. While we can add the language strings, fully supporting RTL UI rendering will require a substantial refactoring of our layout system. We have recorded this request, but it will be a long-term goal.

---

是的，APITable 支持国际化 (i18n)。不过，阿拉伯语是一种从右到左 (RTL) 书写的语言，这在前端 UI 布局上是一个非常大的技术挑战。虽然我们可以增加语言翻译文件，但要完全支持 RTL UI 渲染，需要对我们的底层布局系统进行大幅重构。我们已记录此需求，但这是一个长期的技术难点，可能需要较长时间来实现。
